tests/Jwt/FirebaseParserTest.php

```diff
@@ -51,32 +50,15 @@ protected function setUp()
         ];
     }
 
+    /**
+     * @expectedException \Equip\Auth\Exception\InvalidException
```

Comment:
That is incorrect, it has existed [since v3.2](https://github.com/sebastianbergmann/phpunit/blob/4.8.0/src/Framework/TestCase.php#L448-L455)